### ISSUE TITLE
Add QUnit test for conversion of WebP images

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#674]
Following on from https://github.com/kiwix/kiwix-js/pull/670#discussion_r527896572, once #672 is complete, we should add a QUnit test for WebP image reading:

> It would be good to add a QUnit test on feedNodeWithBlob function : it could create a DOM image element, call this function with a webp content (like the sample image with 2 pixels height), and the check its height afterwards.
This might probably need to refactor this function a bit : add a callback parameter or make it return a Promise, so that the unit test can know when the image has been decoded.

**Comment by kelson42** [comment]
The value of auch a unit test decreases rapidly with time because les and less people uses a Browser without native webp support.

**Comment by mossroy** [comment]
This should be superseded by #748 that should have a UI test that checks that an image is displayed (and check its dimensions).
If we keep running this scenario on browsers that do not support webp (IE11, Firefox 45), it will check that conversion of webp images work.
I agree with you that it would be enough.
